raiden_contracts/utils/bytecode.py

```diff
@@ -5,7 +5,6 @@ def runtime_hexcode(contracts_manager, name, length):
         name: name of the contract such as CONTRACT_TOKEN_NETWORK
         length: the length of the runtime code
     """
-    compiled_bytecode = contracts_manager.contracts[name]['bin']
-    compiled_bytecode = compiled_bytecode[-length:]
+    compiled_bytecode = contracts_manager.contracts[name]['bin-runtime']
     compiled_bytecode = hex(int(compiled_bytecode, 16))
```

Comment:
I believe this `hex(int(...))` line can also be removed.

Reply:
Or not.  Maybe the initial `0x` is missing or so.

Reply:
`web3.eth.getCode(address)` returns `HexBytes('0x..')`, so you would still have to cast.